core/eolearn/core/utils/parsing.py

```diff
@@ -139,9 +166,12 @@ def _parse_dict(self, features: dict) -> List[Union[Tuple[FeatureType, str, str]
         return feature_specs
 
     def _parse_sequence(
-        self, features: Sequence
+        self,
+        features: Union[
+            Tuple[FeatureType, str], Tuple[FeatureType, str, str], Tuple[FeatureType, EllipsisType], SequenceFeatureSpec
```

Comment:
This makes me think if it would make sense to define i.e. 
- `SimpleFeatureSpec = Literal[FeatureType.TIMESTAMP, FeatureType.BBOX]`
- `TupleFeatureSpec = Tuple[FeatureType, str], Tuple[FeatureType, str, str]` 
- `(Single)FeatureSpec = Union[SimpleFeatureSpec, TupleFeatureSpec]`

which could then be used throughout the code

Reply:
Yeah, but imagine having to come up with all the names :see_no_evil:  was also thinking of a similar approach, maybe i should try

Reply:
Dick way of saying you don't like the names I proposed!

Reply:
Feel free to resolve if this is outdated or if you don't plan on making any further changes related to this.